src/modules/storefront/components/Accordions.tsx

```diff
@@ -0,0 +1,75 @@
+import { CSSProperties } from 'react';
+
+import { ReactComponent as ArrowDownIcon } from '../../shared/assets/icons/chevronDownOutlined.svg';
+import { AccordionsData, SpecificContentAccordion } from '../interfaces';
+import './Accordions.css';
+
+export const Accordions = ({ data }: { data: AccordionsData }) => {
+  const { styleData, contentData } = data;
+
+  return (
+    <div className="pw-flex pw-flex-col pw-gap-8 pw-px-2 sm:pw-px-0">
```

Comment:
Inves de colocar o container no accordion, coloca ele em uma div superior a essa, tendo em vista que no futuro pode ter um background color para essa parte sem contar que se mexerem no flex provavlemnte vai dar ruim no container e na posiçao dele

Reply:
isso aqui ainda pode dar problema no futuro
```suggestion
<div style={{bakgroundColor: bla bla}}>
  <div className="pw-container pw-mx-auto">
      <div className="pw-flex pw-flex-col pw-gap-8 pw-px-2 sm:pw-px-0">
```